content/metadata.yaml

```diff
@@ -171,8 +171,11 @@ authors:
     initials: MDK
     orcid: 0000-0003-1258-5221
     email: mkessl11@jhmi.edu
+    funders:
+        - National Institutes of Health (R01DE027809-01A1S1)
     affiliations:
-      - Department of Oncology, Johns Hopkins University
+      - Department of Oncology, Johns Hopkins University, Baltimore, MD 21287
```

Comment:
Not sure we need the zip code here. I think we're going to make a zip field. For now, I suggest keeping just the names.